packages/components/psammead-paragraph/src/index.stories.jsx

```diff
@@ -13,4 +13,11 @@ storiesOf('Paragraph', module)
       <Paragraph script={script}>{paragraph}</Paragraph>
     )),
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'without script',
```

Comment:
As `script` is now optional, I think it would make more sense to me if this was named `default`, and then this would be named `with script`:

https://github.com/bbc/psammead/blob/00be6dcaa1b72a492a620428f23fa1eb629641e9/packages/components/psammead-paragraph/src/index.stories.jsx#L10-L16